src/services/swap_service.ts

```diff
@@ -154,6 +160,20 @@ export class SwapService {
             attributedSwapQuote,
         );
 
+        // set the allowance target based on version. V0 is legacy param to support Nuo integrator
+        let allowanceTarget;
+        switch (swapVersion) {
+            case SwapVersion.V0:
+                allowanceTarget = this._contractAddresses.forwarder;
```

Comment:
If the sellToken is `ETH` the allowance target is `NULL_ADDRESS`, i.e no allowances needed to sell ETH via the Forwarder.

If the sellToken is any ERC20Token, then the allowanceTarget is `erc20Proxy`.

There are no cases where you need to set the allowance on the forwarder.